projects/batfish-client/src/main/java/org/batfish/client/BfCoordWorkHelper.java

```diff
@@ -47,7 +47,7 @@ public BfCoordWorkHelper(String workMgr, BatfishLogger logger, Settings settings
     _logger = logger;
     _settings = settings;
     try {
-      _client = getClientBuilder().build();
+      _client = getClientBuilder().build().register(BfClientRequestFilter.class);
```

Comment:
is this really the right place to put this? Why not in the getClientBuilder() command, for example?